index.html

```diff
@@ -451,11 +451,13 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
         <b class="header">each</b><code>_.each(list, iteratee, [context])</code>
         <span class="alias">Alias: <b>forEach</b></span>
         <br />
-        Iterates over a <b>list</b> of elements, yielding each in turn to an <b>iteratee</b>
-        function. The <b>iteratee</b> is bound to the <b>context</b> object, if one is
+        Iterates over a <b>list</b> of elements, yielding each in turn to an
+        <a href="#iteratee"><b>iteratee</b></a> function.
```

Comment:
Let's just leave this as is for now. Bold __iteratee__ to indicate that it's a variable name, but not linked since it's not transformed through `_.iteratee`. We can revisit this wording in another PR.